src/resmom_win/start_exec.c

```diff
@@ -2258,10 +2231,18 @@ finish_exec(job *pjob)
 			}
 			si.hStdInput = hReadPipe_dummy;
 		}
-		if (script_out != -1)
+		if (script_out != -1) {
 			si.hStdOutput = (HANDLE)_get_osfhandle(script_out);
-		if (script_err != -1)
+			if (si.hStdOutput == -1) {
+				log_err(-1, __func__, "Unable to get handle for the script output file");
```

Comment:
Use errno here, for reference 
https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/get-osfhandle?view=vs-2019